src/components/dashboard/FaceVerification/hooks/useFaceTecSDK.js

```diff
@@ -23,9 +23,15 @@ const log = logger.child({ from: 'useFaceTecSDK' })
  * @return {void}
  */
 export default eventHandlers => {
-  // Configuration callbacks refs
+  // parse options
   const { onInitialized = noop, onError = noop } = eventHandlers || {}
-  const accessors = useRealtimeProps([onInitialized, onError])
+
+  // state vars
+  const [initialized, setInitialized] = useState(false)
+  const [lastError, setLastError] = useState(false)
+
+  // Configuration callbacks refs
+  const accessors = useRealtimeProps([onInitialized, onError, setInitialized, setLastError])
```

Comment:
any variable which changing during lifecycle should be passed to function using Zoom API via  useRealtimeProps  middleware. it uses refs inside and returns non-mutable functions. so funciton/effect don't require deps at all. it could simple call accessors.someCallback() or get the latest prop value by calling accessor.getSomeProp()